File: C/1006.c
#include <stdio.h>
 
int main() {
 
    float a,b,c, media;
    scanf("%f", &a);
    scanf("%f", &b);
    scanf("%f", &c);
    media = a * 2;
    media = media + b * 3;
    media = media + c * 5;
    media = media / 10;
    printf("MEDIA = %.1f\n", media);
}

File: Java/1006.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double a = scanner.nextDouble();
        double b = scanner.nextDouble();
        double c = scanner.nextDouble();
        double d = ((a*2) + (b*3) + (c*5))/10;
        System.out.printf("MEDIA = %.1f\n", d);
    }
}
